config/betterquesting/DefaultQuests/Quests/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==/Magnetohydrodyna-52Yelo6KSLGumzXo-_F5DQ==.json

```diff
@@ -0,0 +1,52 @@
+{
+  "questIDHigh:4": -1772695771338356559,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": -2640443013906741533,
+      "questIDLow:4": -6897090588793529674
+    }
+  },
+  "questIDLow:4": -5865034814944347891,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 1,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "gregtech:gt.metaitem.01",
+        "Count:3": 1,
+        "Damage:2": 11583,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Magnetohydrodynamically Constrained Starmatter",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Or MHDCSM for short.\n\nThis is your primary metal for UXV components. It’s tricky to make and extremely expensive, as all endgame materials should be.\n\nInstead of solidifying it directly, you need to coat Eternity parts with it.\n\nHave fun!"
```

Comment:
Is MHDCSM a metal? I'd just say material.

Reply:
resolved